components/Tabs/src/MDCTabBar.m

```diff
@@ -78,6 +79,10 @@ static MDCItemBarAlignment MDCItemBarAlignmentForTabBarAlignment(
   return MDCItemBarAlignmentLeading;
 }
 
+static inline UIColor *kRippleColor() {
```

Comment:
Static methods do not have `k` prefixes; `k` prefixes are reserved for constants. `RippleColor` is the preferred name in this case.